julec: improve consistency in `julec` and `julec help`

koibtw opened the pull request:
changes:
-refactored the code to use the `help()` instead of hard-coding commands when invoking `julec` with no arguments
-changed grammar to be more consistent

Signed-off-by: Adam Perkowski <adas1per@protonmail.com>

mertcandav commented on src/julec/main.jule:
```diff
@@ -170,20 +171,14 @@ fn processCommand(&args: []str): bool {
+
+Commands:`)
+	help(args, 4)
+	println(`Compilation:
```
```suggestion
	println(`
Compilation:
```

mertcandav reviewed (changes requested):
It looks good in general.
But please keep the new-line between commands.

koibtw committed:
compiler: improve consistency in `julec` and `julec help`

changes:
-refactored the code to use the `help()` instead of hard-coding commands when invoking `julec` with no arguments
-changed grammar to be more consistent

Co-authored-by: mertcandav <mertcandav@outlook.com>
Signed-off-by: Adam Perkowski <adas1per@protonmail.com>

mertcandav reviewed (approved):
LGTM. Thanks for your contribution.
